### ISSUE TITLE
props.onRenderDay is filtered by asConfig

### ISSUE THREAD

**Issue by magegu** [zippyui/react-date-picker#17]
I was not able to use the onRenderDay property. So i digged into the src and found out that the props.onRenderDay is not passed to the day that should handle the event. It is filtered by the asConfig(props) method. 

```
console.log("before:",  props.onRenderDay) // ALL GOOD
var viewProps = props
var viewProps = asConfig(props) // THIS FILTERS THE onRenderDay
console.log("after:",  viewProps.onRenderDay) // UNDEFINED
viewProps.onRenderDay = props.onRenderDay // QUICKFIX
```
Found [here](https://github.com/zippyui/react-date-picker/blob/master/dist/react-date-picker.js#L231)

**Comment by radubrehar** [comment]
Thanks for the bug report. Will come up with a new release with the fix either tomorrow or on Monday.

Hope you find the component useful!

**Comment by magegu** [comment]
thanks, yes. the component is usefull :-)  I'll post some suggestions later!

**Comment by radubrehar** [comment]
I've just published 2.0.10 on npm with a fix for this issue. We will soon add automated testing and continuous integration in order to make sure regressions like this are highly unlikely.

Looking forward to your suggestions so we can make react-date-picker the best date picker around!
